Solutions/Azure Logic Apps/Package/mainTemplate.json

```diff
@@ -0,0 +1,392 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "author": "Microsoft support@microsoft.com",
```

Comment:
1. Please add a hyphen and should look like `Microsoft - support@microsoft.com`
2. There are 4 places in this file which are missing hyphen in `Microsoft - support@microsoft.com`